packages/slice-machine/lib/builders/common/Zone/Card/components/NewField.js

```diff
@@ -54,6 +58,30 @@ const NewField = ({ widgetTypeName, fields, onSave, onCancelNewField }) => {
     }
   }, [fieldRef]);
 
+  const handleLabelChange = (e, values, setValues) => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
```

Comment:
how is this an linter error?

Reply:
wait... it's still using JS,

Would it be possible to migrate this to tsx?

Reply:
converted!